### DISCUSSION TITLE
MQTT - JSON format.

### DISCUSSION THREAD

**Discussion by dgoadby** [arendst/Tasmota#18089] [Q&A]
I am using five Xiaomi Bluetooth temperature/humidity units with a Tasmota unit. I am post-processing the MQTT messages which are in JSON format and then insert the data into Influxdb.

I get three SENSOR packets with two sensor readings in the first two and one in the remaining one.  

To make post-processing a little easier it would be nice if the MQTT data was just one sensor per packet. In my case I would be expecting five packets instead of three.

Is there any way to telling Tasmota to only send one sensor per MQTT packet? If not, then it would be a nice feature to have.

**Comment by sfromis** [comment]
The various `MI32Option` settings influence how sensor data is published. Try what works for you.
https://tasmota.github.io/docs/Bluetooth_MI32/#commands

Those will have to be repeated after each boot, like being in the file `autoexec.bat`
